src/main/java/com/foodybuddy/service/impl/CityServiceImpl.java

```diff
@@ -0,0 +1,289 @@
+package com.foodybuddy.service.impl;
+
+import com.foodybuddy.dao.CityDAO;
+import com.foodybuddy.dao.StateDAO;
+import com.foodybuddy.dao.impl.CityDAOImpl;
+import com.foodybuddy.dao.impl.StateDAOImpl;
+import com.foodybuddy.model.City;
+import com.foodybuddy.model.State;
+import com.foodybuddy.service.CityService;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.TransactionException;
+import org.springframework.stereotype.Service;
+
+/**
+ * The Class CityServiceImpl.
+ */
+@Service("cityService")
+public class CityServiceImpl implements CityService {
+
+	/** The Transaction failure exception. */
+	TransactionException TransactionFailureException = new TransactionException(
```

Comment:
Why are you creating this object beforehand. This has to live for the whole duration of service class.
Instead creating the object when required would be better